### PR TITLE
Beam wallet and fixes in CLI

### PR DESCRIPTION
**Pull request by kasiaturbiarz**

### CODE REVIEW

File: src/Nethermind/Nethermind.BeamWallet.Tests/JsonRpcWalletClientProxyTests.cs [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+{
+    public class JsonRpcWalletClientProxyTests
+    {
+        private IJsonRpcWalletClientProxy _jsonRpcWalletClientProxy;
+        
+        [SetUp]
+        public void Setup()
+        {
+            var urls = new[] {"http://localhost:8545/"};
+            var jsonRpcClientProxy=  new JsonRpcClientProxy(new DefaultHttpClient(new HttpClient(),
```
Comment (tkstanczak) [inline src/Nethermind/Nethermind.BeamWallet.Tests/JsonRpcWalletClientProxyTests.cs:21]: happy to see tests

File: src/Nethermind/Nethermind.BeamWallet/Modules/Addresses/AddressesModule.cs [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using Terminal.Gui;
+
+namespace Nethermind.BeamWallet.Modules.Addresses
+{
+    internal class AddressesModule : IModule
+    {
+        private static readonly Regex _urlRegex = new Regex(@"^http(s)?://([\w-]+.)+[\w-]+(/[\w- ./?%&=])?",
+            RegexOptions.Compiled);
```
Comment (tkstanczak) [inline src/Nethermind/Nethermind.BeamWallet/Modules/Addresses/AddressesModule.cs:28]: good to see it compiled

File: src/Nethermind/Nethermind.BeamWallet/Modules/Data/DataModule.cs [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+            }
+        }
+
+        private static decimal WeiToEth(string result) => (decimal.Parse(result) / 1000000000000000000);
+
+        private static IEnumerable<Token> InitTokens()
+        {
+            var list = new[]
+            {
+                new Token("DAI", "0x6b175474e89094c44da98b954eedeac495271d0f"),
```
Comment (tkstanczak) [inline src/Nethermind/Nethermind.BeamWallet/Modules/Data/DataModule.cs:147]: these are mainnet or Goerli addresses?

File: src/Nethermind/Nethermind.Runner/configs/mainnet_beam.cfg [hunk 4]
```diff
@@ -16,7 +16,7 @@
     "ActivePeersMaxCount": 100
   },
   "JsonRpc": {
-    "Enabled": false,
+    "Enabled": true,
```
Comment (tkstanczak) [inline src/Nethermind/Nethermind.Runner/configs/mainnet_beam.cfg:19]: you committed this change by mistake

File: src/Nethermind/Nethermind.BeamWallet/Modules/Data/DataModule.cs (hunk 3 above)
Comment (kasiaturbiarz) [reply src/Nethermind/Nethermind.BeamWallet/Modules/Data/DataModule.cs:147]: mainnet

**Pull request merged**